homeassistant/components/kodi/media_player.py

```diff
@@ -307,7 +311,7 @@ def _no_active_players(self):
         return not self._players
 
     @callback
-    def async_on_speed_event(self, sender, data):
+    def async_on_speed_event(self, sender, data):  # pylint: disable=unused-argument
```

Comment:
Type annotations here too. Also, maybe we can use underscores in the names instead of pylint comments?

Reply:
My laptop battery is dead soon. So I'll continue tomorrow. When I change the names, jsonrpc is complaining unfort.:
```
2022-11-04 18:34:21.486 ERROR (MainThread) [jsonrpc_base.jsonrpc] KodiBinaryEntity._async_off() got an unexpected keyword argument 'sender'
Traceback (most recent call last):
  File "/home/efelon/dev/core/venv/lib/python3.10/site-packages/jsonrpc_base/jsonrpc.py", line 92, in async_receive_request
    result = handler(*args, **kwargs)
TypeError: KodiBinaryEntity._async_off() got an unexpected keyword argument 'sender'
```